scr/main/java/com/globant/models/User.java

```diff
@@ -0,0 +1,114 @@
+package com.globant.models;
+
+public class User {
+
+	private String username;
+
+	private String password;
+
+	private String name;
+
+	private boolean logged;
+
+	private ShoppingCart shoppingCart;
+
+	public User(String username, String name, String password) {
+
+		this.username = username;
+		this.password = password;
+		this.name = name;
+		this.logged = false;
+		this.shoppingCart = new ShoppingCart(username);
+	}
+
+	/**
+	 * Getter method for username parameter.

```

Comment:
The javadocs in getters and setters are redundant, it's better to add a single javadoc per each attribute and leave the getters and setters without javadocs (except when there is additional logic)